main.go

```diff
@@ -280,54 +278,70 @@ func (s *Session) firstNav(ctx context.Context) error {
 	return nil
 }
 
-// navToEnd waits for the page to be ready to receive scroll key events, by
-// trying to select an item with the right arrow key, and then scrolls down to the
-// end of the page, i.e. to the oldest items.
+// navToEnd waits for the page to be ready to receive scroll key events,
+// and then scrolls down to the end of the page, i.e. to the oldest items.
 func navToEnd(ctx context.Context) error {
-	// wait for page to be loaded, i.e. that we can make an element active by using
-	// the right arrow key.
-	for {
-		chromedp.KeyEvent(kb.ArrowRight).Do(ctx)
-		time.Sleep(tick)
-		var ids []cdp.NodeID
-		if err := chromedp.Run(ctx,
-			chromedp.NodeIDs(`document.activeElement`, &ids, chromedp.ByJSPath)); err != nil {
-			return err
-		}
-		if len(ids) > 0 {
-			if *verboseFlag {
-				log.Printf("We are ready, because element %v is selected", ids[0])
-			}
-			break
-		}
-		time.Sleep(tick)
-	}
+	// wait for page to be loaded
+	chromedp.WaitReady("body", chromedp.ByQuery).Do(ctx)
 
```

Comment:
I'm surprised we can get away with simply removing all that, as it seemed to me the pagedown and end event were getting lost into oblivion if I didn't wait for the page to be loaded before sending them.
But I'll try it out and will let you know.

Reply:
Yes, but notice that the `kb.PageDown,kb.End` are in the `for` loop, so they will happen as needed. 
The part that was removed (`kb.ArrowRight` and wait for `document.activeElement`) we're the parts removed because they no longer had any effect after scrolling.